src/collective/cover/tiles/calendar.py

```diff
@@ -169,12 +177,10 @@ def isToday(self, day):
             self.now[2] == day and self.now[1] == self.month and self.now[0] == self.year)
 
     def getReviewStateString(self):
-        states = self.calendar.getCalendarStates()
-        return ''.join(map(lambda x: 'review_state={0}&amp;'.format(self.url_quote_plus(x)), states))
+        return 'review_state=published&amp;'
 
     def getEventTypes(self):
-        types = self.calendar.getCalendarTypes()
-        return ''.join(map(lambda x: 'Type={0}&amp;'.format(self.url_quote_plus(x)), types))
+        return 'Type=Event&amp;'
```

Comment:
@hvelarde I didn't finish the changes.  My first step is to remove `portal_calendar`, after I'll refactor the code and these methods will be removed.